lib/tasks/question_tasks.rake

```diff
@@ -0,0 +1,7 @@
+desc 'close old questions'
+task close_old_questions: :environment do
+  questions = Question.to_be_closed
+  questions.each do |question|
+    question.update_attributes(closed: true)
+  end
```

Comment:
We can rewrite it as `Question.to_be_closed.update_all(closed: true)` to execute only one query.